js/core/version.js

```diff
@@ -1 +1 @@
-module.exports = "19.1.0"; /* DevExpress.VERSION */
+module.exports = "0.0.0";
```

Comment:
Can we use a more specific placeholder (smth like "%VERSION%", "#VERSION_TO_REPLACE#", "{VERSION_PLACEHOLDER}", "$VERSION_DUMMY$", etc) to make this magic constant more clear and avoid unwanted potential replacements? 🤔

Reply:
I like the pattern. We need no non-searchable regular expression/